terraform/main.tf

```diff
@@ -23,18 +23,24 @@ resource "google_compute_subnetwork" "honeypot_subnet" {
   network       = google_compute_network.main.self_link
 }
 
-# standard firewall settings
-resource "google_compute_firewall" "firewall_standard_rule" {
-  name    = "firewall-standard-rule"
```

Comment:
removed `firewall-` prefix because it is obscures the meaning and is unnecessary for google cloud console
![image](https://user-images.githubusercontent.com/33207565/175775854-a3be821a-b6fd-496d-89a5-04dddf8a5bc4.png)